### ISSUE TITLE
Budget related entries in Management shouldn't appear if budgets are globally disabled

### ISSUE THREAD

**Issue by periflo90** [consul/consul#1901]
We've found something interesting in the Management panel. There are many entries in the left sidebar which reference budgets. For the time being, we have globally disabled the budgets feature.

Having disabled it, when accessing a budget-related feature, it gives the following Rails error:

```
FeatureFlags::FeatureDisabled in Management::BudgetsController#create_investments
Feature disabled: budgets
```

I suggest those sidebar entries not to appear if budgets are globally disabled, because the product may seem weak to the end user.

**Comment by periflo90** [comment]
I could try. I'll keep you posted.

**Comment by periflo90** [comment]
Do you mean disabling them as in checking other entries which should throw that error and making them throw it, or checking if the feature is enabled and if false not rendering the menu items?

**Comment by aitbw** [comment]
From what I can understand, you should check if the feature is enabled and if `false`, don't render "Budgets" on the menu, @periflo90.

Feel free to see #1903 to see how this can be achieved in full.

**Comment by periflo90** [comment]
Okay, done. I just made a pull request. Hope everything's okay.

**Comment by bertocq** [comment]
Closed via #1941 👏
